admin/class-boldgrid-backup-admin-jobs.php

```diff
@@ -239,6 +280,7 @@ public function run() {
 		// If there are no jobs or already running, then abort.
 		if ( empty( $this->jobs ) || $this->is_running() ) {
 			wp_die();
+			$this->maybe_fixed_stalled();
```

Comment:
If you add this call immediately after a `wp_die()` call, it will never be ran.

I recommend adding the call after https://github.com/BoldGrid/boldgrid-backup/blob/2e4f70db3b18093d2dc5725f0aed8afb10c0cc57/admin/class-boldgrid-backup-admin-jobs.php#L329

After we get the jobs from the database, we should then maybe fix any stalled ones.